packages/features/form-builder/fieldTypes.ts

```diff
@@ -105,7 +105,8 @@ const configMap: Record<FieldType, Omit<z.infer<typeof fieldTypeConfigSchema>, "
     value: "textarea",
     isTextType: true,
     supportsLengthCheck: {
-      maxLength: 4000,
+      // Keep it as small as possible. It is easier to change to a higher value but coming back to a lower value(due to any reason) would be problematic for users who have saved higher value.
+      maxLength: 1000,
```

Comment:
@hariombalhara Can we use a shared constant here instead of a direct value? Bringing this up since you mentioned it is constant to our code base.

Reply:
I think we can define a local HARD_MAX_LENGTH_TEXTAREA locally in this module. 

It isn't a constant to be used throughout the app now.

Reply:
> I think we can define a local HARD_MAX_LENGTH_TEXTAREA locally in this module.

Yes this would be better I believe

Reply:
I'll take care of that @hariombalhara